R/utils.R

```diff
@@ -1,11 +1,20 @@
 #'@importFrom httr stop_for_status GET user_agent content status_code
 pv_query <- function(params, reformat = TRUE, ...){
+  if(length(params) > 1){
+    data <- lapply(as.list(params), pv_query, reformat = reformat)
+    if(reformat == TRUE){
+      return(do.call(rbind, data))
+    } else {
+      return(data)
```

Comment:
This seems pretty confusing in practice. Absent reformatting the structure is suddenly an extra degree nested. Documentation will need to be a lot more expansive and I'm not sure how I feel about the change.